rfcs/0xx-tei_adapter/README.md

```diff
@@ -0,0 +1,93 @@
+# TEI Adapter
+
+## What is TEI
+## How we do it
+### Do we need an adapter?
+
+The TEI files are in a [GitHUb Repository](https://github.com/wellcomecollection/wellcome-collection-tei). 
+At the time of writing in April 2021 there's about 500(?) TEI files.
+We need to be able to propagate updates as they happen through to the API, and we also need to be able to
+reindex when we make changes in the pipeline. 
+
+GitHub is a relatively reliable and low latency source, so we could in theory just send the changes to a TEI transformer 
+and do a reindex by cloning the repository and/or crawling the TEI files. 
+There are three disadvantages to this approach:
+- We need a "special" reindexer for TEI.
+- We can't keep track of deleted files. If something is deleted, the change will go through the pipeline once, 
+  but there's no way of "remembering" it.
+- We need to assign versions to works resulting from TEI files in order to ensure subsequent updates are 
+  processed correctly in the pipeline. This versioning logic would have to be added explicitly to the 
+  TEI transformer whereas having an adapter and an intermediate store would allow us to use the already existing versioning
+  logic in the versioned store library.
```

Comment:
one more reason: currently we use the # of works in the source table to track the success of a reindex.

50 records in => 50 records out

You can't do that if you don't have a canonical store.

Reply:
Added thanks